### ISSUE TITLE
CMakeDependency('OpenSSL', env, dict(kwargs, modules=['OpenSSL::Crypto', 'OpenSSL::SSL'])) / README.md needs a tweak

### ISSUE THREAD

**Issue by sscotter** [systemd/systemd-netlogd#114]
Hi,

When bulidling on a Debian 12 machine following the instructions in the README.md ...

```
sudo apt install build-essential gperf libcap-dev libsystemd-dev pkg-config meson python3-sphinx
make
sudo make install 
```

... I get the following error ...

```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/mesonbuild/dependencies/detect.py", line 113, in find_external_dependency
    d = c()
        ^^^
  File "/usr/lib/python3/dist-packages/mesonbuild/dependencies/misc.py", line 709, in <lambda>
    cmake_class=lambda name, env, kwargs: CMakeDependency('OpenSSL', env, dict(kwargs, modules=['OpenSSL::Crypto', 'OpenSSL::SSL'])),
                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/mesonbuild/dependencies/cmake.py", line 127, in __init__
    raise DependencyException(msg)
mesonbuild.dependencies.base.DependencyException: CMake binary for machine 1 not found. Giving up.
```

Installing the `libssl-dev` package allows the packages to build successfully.

TL;DR - tweak README.md from ...

```
sudo apt install build-essential gperf libcap-dev libsystemd-dev pkg-config meson python3-sphinx
```
... to ....

```
sudo apt install build-essential gperf libcap-dev libsystemd-dev pkg-config meson python3-sphinx libssl-dev
```

**Comment by adamramadhan** [comment]
+1
